PopPUNK/assign.py

```diff
@@ -389,7 +404,34 @@ def main():
     from .utils import setupDBFuncs
 
     # Dict of QC options for passing to database construction and querying functions
-    qc_dict = {'run_qc': False }
+    if args.length_sigma is None and None in args.length_range and args.prop_n is None \
+        and args.upper_n is None:
+        qc_dict = {'run_qc': False }
+    else:
+        # define defaults if one QC parameter given
+        # length_sigma
+        if args.length_sigma is not None:
+            length_sigma = args.length_sigma
+        elif None in args.length_range:
+            length_sigma = 5 # default used in __main__
+        else:
+            length_sigma = None
+        # prop_n
+        if args.prop_n is not None:
+            prop_n = args.prop_n
+        elif args.upper_n is None:
+            prop_n = 0.1 # default used in __main__
+        else:
+            prop_n = None
```

Comment:
This should be made into a function and combined with the one in __main__

Reply:
Done in previous commit.